app/host/astro_prost/associate.py

```diff
@@ -442,9 +486,7 @@ def associate_transient(
                     )
 
                 # For some reason the value of "verbose" is ignored here, and the effective
-                # level returned by logger.getEffectiveLevel() is 10 (DEBUG)
```

Comment:
@djones1040 Looks like you truncated my comment. You can delete the whole thing if you don't think it's useful.

Reply:
oh I replaced the files entirely w/ alex's recent updates, sorry about that.  wasn't intentional